examples/developer-tools/src/toolbox.ts

```diff
@@ -0,0 +1,28 @@
+export const toolbox = {
+  'kind': 'categoryToolbox',
+  'contents': [
+    {
+      'kind': 'category',
+      'name': 'Input',
+      'categorystyle': 'logic_category',
```

Comment:
If you're going to use the category style might as well use themes on the blocks. That way they stay in sync.